protokube/pkg/protokube/etcd_manifest.go

```diff
@@ -30,70 +32,51 @@ func BuildEtcdManifest(c *EtcdCluster) *v1.Pod {
 	pod.Kind = "Pod"
 	pod.Name = c.PodName
 	pod.Namespace = "kube-system"
+	pod.Labels = map[string]string{"k8s-app": c.PodName}
 
-	pod.Labels = map[string]string{
-		"k8s-app": c.PodName,
-	}
-
-	// TODO another hardcoded version
-	image := "/etcd:2.2.1"
-	imageRegistry := "gcr.io/google_containers"
+	etcdImage := "/etcd:2.2.1"
+	etcdRegistry := "gcr.io/google_containers"
 
-	// Test to determine if the container registry has been passed in as a flag.
-	// If so use the provider registry location.
+	// @check if the container is being overloaded via flags
 	if c.ImageSource == "" {
-		image = imageRegistry + image
+		etcdImage = etcdRegistry + etcdImage
 	} else {
-		image = strings.TrimSuffix(c.ImageSource, "/") + image
+		etcdImage = strings.TrimSuffix(c.ImageSource, "/") + etcdImage
 	}
 
 	pod.Spec.HostNetwork = true
-
 	{
 		container := v1.Container{
 			Name:  "etcd-container",
-			Image: image,
+			Image: etcdImage,
 			Resources: v1.ResourceRequirements{
 				Requests: v1.ResourceList{
 					v1.ResourceCPU: c.CPURequest,
 				},
 			},
-			Command: []string{
-				"/bin/sh",
-				"-c",
-				"/usr/local/bin/etcd 1>>/var/log/etcd.log 2>&1",
-			},
-
-			// Note that we listen on 0.0.0.0, not 127.0.0.1, so we can support etcd clusters
-			Env: []v1.EnvVar{
-				{Name: "ETCD_NAME", Value: c.Me.Name},
-				{Name: "ETCD_DATA_DIR", Value: "/var/etcd/" + c.DataDirName},
-				{Name: "ETCD_LISTEN_PEER_URLS", Value: fmt.Sprintf("http://0.0.0.0:%d", c.PeerPort)},
-				{Name: "ETCD_LISTEN_CLIENT_URLS", Value: fmt.Sprintf("http://0.0.0.0:%d", c.ClientPort)},
-				{Name: "ETCD_ADVERTISE_CLIENT_URLS", Value: fmt.Sprintf("http://%s:%d", c.Me.InternalName, c.ClientPort)},
-				{Name: "ETCD_INITIAL_ADVERTISE_PEER_URLS", Value: fmt.Sprintf("http://%s:%d", c.Me.InternalName, c.PeerPort)},
-				{Name: "ETCD_INITIAL_CLUSTER_STATE", Value: "new"},
-				{Name: "ETCD_INITIAL_CLUSTER_TOKEN", Value: c.ClusterToken},
-			},
+			Command: []string{"/usr/local/bin/etcd"},
```

Comment:
I don't disagree, but these logs tend to get rotated more often than we would like.  I agree that it is annoying that they don't appear in the k8s logs.  I'm not sure if `tee` is in this image, but if it is we can do this trick: https://github.com/kubernetes/kops/blob/1eb9949051612e60874a815ca203f85ac8d512b7/nodeup/pkg/model/kubeproxy.go#L133 to log to both places...